docs/setup-cluster/basic-setup-guide/basic.rst

```diff
@@ -0,0 +1,56 @@
+.. _basic-setup:
+
+.. _install-cluster:
+
+#############
+ Basic Setup
+#############
+
+If you have not used Determined before, and you want to quickly set up a new training environment,
+you are at the right place!
+
+.. note::
+
+   To set up a Determined training environment on-prem or on cloud, visit :ref:`Advanced Setup
+   <advanced-setup>`.
+
+
+**Prerequisites**
+
+-  :ref:`Installation Requirements <system-requirements>`
+
+*********************************
+ Step 1 - Install Docker Desktop
```

Comment:
"Docker Desktop" is the mac-specific software.  This should probably just be called "Install Docker".